clients/python-wrapper/lakefs/reference.py

```diff
@@ -76,10 +76,12 @@ def objects(self) -> ObjectManager:
     @staticmethod
     def _get_generator(func, *args, max_amount: Optional[int] = None, **kwargs):
         has_more = True
+        after = ""
         with api_exception_handler():
             while has_more:
-                page = func(*args, **kwargs)
+                page = func(*args, after=after, **kwargs)
```

Comment:
IMO it makes more sense to update the `after` value inside of the kwargs.
This is not an urgent matter.

Reply:
Just resolved by you 🙂